corehq/apps/app_manager/dbaccessors.py

```diff
@@ -22,14 +22,14 @@ def domain_has_apps(domain):
     return len(results) > 0
 
 
-def get_latest_released_app_doc(domain, app_id, min_version=None):
+def get_latest_released_app_doc(domain, app_id):
```

Comment:
@dannyroberts I think I'm with you for most of the comment, but I don't follow the business about min_version changing things.  As @millerdev suggested, this function is only called with the `min_version` parameter in that one place below.

Example:
working copy is v8
unreleased build at v6
released builds at v3, v5, v7

Scenario 1:
`get_app` is called with the working copy `app_id`.  `min_version` is then set to -1 and effectively isn't used.

Scenario 2:
`get_app` is called with the id of build 7 (unreleased).  `min_version` is set to -1

Scenario 3:
`get_app` is called with the id of build 5 (released).
`get_latest_released_app_doc` is called with the `app_id` (of the working copy), and `min_version=5`.
That parameter tells `get_latest_released_app_doc` to not return anything with a version before 5, but since we know version 5 exists (since that's how we got here), it wouldn't do that anyways.
It will return the biggest release of the set of all releases, and since the version number passed in is always in the set of all releases, it would never use a smaller version number anyways.

I'm pretty convinced this is the case.  Happy to go to voice if there's some edge case I'm missing.